### ISSUE TITLE
bug: IOS bundle resets after closing app.

### ISSUE THREAD

**Issue by HatemsherifMF** [Cap-go/capacitor-updater#362]
### Capacitor Version:

 💊   Capgo Doctor  💊
 
Latest Dependencies:

  @capacitor/cli: 6.0.0
  @capacitor/core: 6.0.0
  @capacitor/android: 6.0.0
  @capacitor/ios: 6.0.0

Installed Dependencies:

  @capacitor/android: 6.0.0
  @capacitor/ios: 6.0.0
  @capacitor/core: 6.0.0
  @capacitor/cli: 6.0.0

### Plugin Version:

●   💊   Capgo Doctor  💊
│  
│
●   OS: darwin Darwin Kernel Version 23.4.0: Fri Mar 15 00:12:41 PDT 2024; root:xnu-10063.101.17~1/RELEASE_ARM64_T8103
│  
│
●   Node: v18.13.0
│  
│
●   Installed Dependencies:
│  
│
●     @capgo/cli: 4.4.5
│
●     @capgo/capacitor-updater: 6.0.2
│
●  
│  
✓   Latest Dependencies:
│
●     @capgo/cli: 4.4.5
│
●     @capgo/capacitor-updater: 6.0.2

### context(s)

```md
ManualModel: true
AutoMode: false
CapgoCloud: false
OnPremise: true
```

### Platform(s)
IOS only

### Current Behavior
Updating bundle is working fine on android, while on ios after zipping using the cli & hosting the file:

1- I use notifyAppReady on main.ts.
2- I download the zip file using download method.
3- i set the bundle using set method & app restarts successfuly.
4- bundle is updated & everything seems to be fine.
5- i close the app and re open it again.
6- i find that the changes of my latest bundle has gone & the initial bundle is active.
7- I also found that plugin still detects the version that i passed is the current version ( which is not true )

### Expected Behavior
I expect the latest successful bundle changes to be applied rather than the initial Bundle.

### Other Technical Details
`npm --version` : 8.19.3

`node --version`: v18.13.0

`pod --version` : 1.15.2

here is a simplified example of the implementation:

**main.ts :**

```
import { CapacitorUpdater } from '@capgo/capacitor-updater';
CapacitorUpdater.notifyAppReady();
```

**app.component.ts:**

```
export class myComponent implements OnInit {
  progress = 0;

  ngOnInit(): void {
    this.addListener();
    this.update();
  }

  async update() {
    const version = await CapacitorUpdater.download({
      url: 'https://example.com/www.zip',
      version: '0.0.2',
    });
    CapacitorUpdater.set(version);
  }

  addListener(): void {
    CapacitorUpdater.addListener('download', (event) => {
      this.progress = event.percent / 100;
    });
  }
}
```

**on the updated bundle I've changed some ui & removed the update & add listener methods to avoid going on an endless loop**

**Comment by HatemsherifMF** [comment]
Was fixed on V6.0.4 https://github.com/Cap-go/capacitor-updater/releases/tag/6.0.4

**Comment by rricamar** [comment]
> Was fixed on V6.0.4 https://github.com/Cap-go/capacitor-updater/releases/tag/6.0.4

Yes, `v6.0.4` is working as expected 👍
